content/tutorial-multisign.md

```diff
@@ -20,9 +20,11 @@ To use multi-signing:
 Availability of Multi-Signing
 -----------------------------
 
-Multi-signing is built into `rippled` starting with [version 0.31.0](https://wiki.ripple.com/Rippled-0.31.0). Because multi-signing is a change in transaction processing, the change is due to be enabled by an **Amendment** to the Ripple Consensus Protocol. This Amendment must be approved by a consensus of validators showing consistent support for the feature over a period of time. For more information, see [Amendments](concept-amendments.html).
+Multi-signing has been enabled by an [**Amendment**](concept-amendments.html) to the Ripple Consensus Protocol since 2016-06-27.
```

Comment:
As we discussed offline, the alternative "was enabled" leaves room for the status to have changed. The phrasing "has been enabled since..." makes it clear that it's the same status now.